### PR TITLE
Fix JSHint W014

### PR DESCRIPTION
**Pull request by 100mik**
Signed-off-by: Soumik Majumder <soumikm@vmware.com>

**Description**

This PR fixes JSHint W014 warning reported by Sonatype Lift [here](https://lift.sonatype.com/result/meshery/meshery/01FHW0QAMP11ERCD7WXRTV9BKV?t=CustomTool%20%22JSHint%22%7CW014)

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/GrafanaCustomChart.js [hunk 1]
```diff
@@ -436,12 +435,10 @@ class GrafanaCustomChart extends Component {
             const newData = [];
 
             // if (typeof cd.labels[datasetInd] === 'undefined' || typeof cd.datasets[datasetInd] === 'undefined'){
-            let legend = typeof target.legendFormat !== 'undefined'
-              ? target.legendFormat
+            let legend = typeof target.legendFormat !== 'undefined' ? target.legendFormat
```
[the remark this answers is not in the document]
Comment (100mik) [reply ui/components/GrafanaCustomChart.js:438]: Making this change as well

File: ui/components/GrafanaCustomChart.js [hunk 2]
```diff
@@ -644,8 +635,7 @@ class GrafanaCustomChart extends Component {
           // cd.datasets[datasetInd].borderColor = panel.sparkline.lineColor;
           // cd.datasets[datasetInd].backgroundColor = panel.sparkline.fillColor;
 
-          const dataLength = chartConfig.data.columns && chartConfig.data.columns.length > 1
-            ? chartConfig.data.columns[1].length
+          const dataLength = chartConfig.data.columns && chartConfig.data.columns.length > 1 ? chartConfig.data.columns[1].length
```
[the remark this answers is not in the document]
Comment (100mik) [reply ui/components/GrafanaCustomChart.js:638]: I believe just 
```js
Array.isArray(ychartConfig.data.columns) && chartConfig.data.columns.length > 1
```
does the job. cause `isArray` would return false if it is undefined/null

**Comment by 100mik** [comment]
Super sorry for the delay! Incorporated the feedback.

**Comment by leecalcote** [comment]
Thanks for seeing this through, @100mik

**Pull request merged**